File: Python/day10.py
n = int(raw_input().strip())
for i in range(n):
    num = int(raw_input().strip())
    blist = []
    while num > 0:
        blist.append(num % 2)
        num >>= 1
    print "".join(map(str, blist[-1::-1]))

File: Ruby/day10.rb
n = gets.strip.to_i
(1..n).each do |i|
    num = gets.strip.to_i
    barray = []
    while num > 0
        barray.push(num % 2)
        num >>= 1
    end
    puts barray.reverse.join("")
end
